internal/provider/resource_azure_cloud_router_connection_test.go

```diff
@@ -17,6 +19,9 @@ func TestAccCloudRouterConnectionAzureRequiredFields(t *testing.T) {
 	crConnAzureResult := testutil.RHclCloudRouterConnectionAzure()
 	var cloudRouterCircuitId, cloudRouterConnectionCircuitId string
 
+
+	return
```

Comment:
Why this return?

Reply:
I'm not sure how this came in, but I threw it back out again